ConwaysGameOfLife.Tests/UniverseShould.cs

```diff
@@ -124,5 +126,17 @@ public void WhenGivenALocation_ReturnsCell()
             Assert.Equal(0, result.Location.Y);
             Assert.Equal(CellState.Dead, result.CellState);
         }
+
+        [Fact]
+        public void TestUniverseHelper_UniversesAreEqual() // TODO: rename method
+        {
+            var gridWidth = 4;
+            var gridLength = 4;
+            var universe = new Universe(gridWidth, gridLength);
+
+            var result = UniverseHelper.UniversesAreEqual(universe, new Universe(4, 4));
+
+            Assert.True(result);
+        }

```

Comment:
Probably worth instantiating the two universes in different ways e.g. one as on line 135 and another with a List of cells similar to the neighbour tests.